source/scriptHandler.py

```diff
@@ -242,8 +250,13 @@ def script(description="", category=None, gesture=None, gestures=None, canPropag
 	@type gesture: string
 	@param gestures: A list of gestures associated with this script
 	@type gestures: list(string)
-	@param canPropagate: Whether this script should also apply for focus ancestors.
+	@param canPropagate: Whether this script should also apply when it belongs to a  focus ancestor object.
 	@type canPropagate: bool
+	@param bypassInputHelp: Whether this script should run when input help is active.
+	@type bypassInputHelp: bool

```

Comment:
The only script which should need to bypass input help is input help, no?

Reply:
Nope. We also have raille_scrollBack, braille_scrollForward and the braille input toggle modifiers that bypass input help.